CONTRIBUTING.md

```diff
@@ -109,6 +109,48 @@ See also:
 
 - [Developing](#developing)
 
+# Working with Unreleased Forc and Fuel-Core Functionality
+
+When you need to work with unreleased versions of `forc` or `fuel-core`, you can specify a git branch instead of a version number.
+
+## Using Unreleased Forc
+
+To use an unreleased version of `forc`:
+
+1. **Edit the `VERSION` File**
+Open the `packages/forc/VERSION` file and replace the version number (e.g., 0.59.0) with the desired git branch:
+
+    ```bash
+    git:some/branch-name
+    ```
```

Comment:
(Can't suggest this change via GH)
<img width="221" alt="Screenshot 2024-10-01 at 15 19 18" src="https://github.com/user-attachments/assets/cc8e6d1e-109b-47e5-b0e8-7022b48890f2">


Un-needed whitespace, and this isn't a bash file. (not 100% it's a txt file either, but seems more appropriate)

Same goes for all the below snippets :)

Reply:
Well now I've learn't something.